README.rst

```diff
@@ -69,6 +69,168 @@ The *apidoc* extension uses the following configuration values:
 
    **Optional**, defaults to ``False``.
 
+Migration from pbr
+------------------
+
+`pbr`_ has historically included a `custom variant <build_sphinx>`_ of the
+``build_sphinx`` distutils command. This provides, among other things, the
+ability to generate API documentation as part of build process. Clearly this is
+not necessary with this extension.
+
+There are two implementations of the API documentation feature in *pbr*:
+*autodoc_tree* and *autodoc*. To describe the difference, let's explore how one
+would migrate real-world projects using both features.
+
+autodoc_tree
+~~~~~~~~~~~~
+
+As *autodoc_tree* is based on *sphinx-apidoc*, migration is easy. Lets take
+`python-openstackclient`_ as an example, looking at minimal versions of
+``setup.cfg`` and ``doc/source/conf.py``:
+
+.. code-block:: ini
+
+   [build_sphinx]
+   all_files = 1
+   build-dir = doc/build
+   source-dir = doc/source
+
+   [pbr]
+   autodoc_tree_index_modules = True
+   autodoc_tree_excludes =
+     setup.py
+     openstackclient/volume/v3
+     openstackclient/tests/
+     openstackclient/tests/*
+   api_doc_dir = contributor/api
+
+.. code-block:: python
+
+   extensions = ['']
+
+Once migrated, this would look like so:
+
+.. code-block:: ini
+
+   [build_sphinx]
+   all_files = 1
+   build-dir = doc/build
+   source-dir = doc/source
+
+.. code-block:: python
+
+   extensions = ['sphinxcontrib.apidoc']
+
+   apidoc_module_dir = '../../openstack'
+   apidoc_excluded_paths = [
+     'volume',
+     'tests'
+   ]
+   apidoc_output_dir = 'contributor/api'
+
+.. note::
+
+    You could also remove the ``[build_sphinx]`` section from ``setup.cfg`` if
+    you wished to build documentation with ``sphinx-build`` directly instead.
+
+There are a couple of changes here:
+
+- Replace ``autodoc_tree_index_modules`` with ``apidoc_module_dir``
```

Comment:
It seems a little weird to call this a 'replacement' when they're in different files?

Reply:
I struggled to word this and this was the best I could come up with. Would welcome suggestions here.

Reply:
It's tricky. I think I would be inclined to just have "remove (autodoc-related options from) the [pbr] section from setup.cfg" as one instruction and then have separate instructions for setting each of the config options in conf.py, e.g. for `apidoc_module_dir` set it to the same value that was previously in `autodoc_tree_index_modules` in setup.cfg.

Reply:
Done